cluster/centos/config-build.sh

```diff
@@ -28,6 +28,9 @@ ETCD_VERSION=${ETCD_VERSION:-"2.2.1"}
 # Define k8s version to use.
 K8S_VERSION=${K8S_VERSION:-"1.1.1"}
 
+# Define docker version to use.
+DOCKER_VERSION=${DOCKER_VERSION:-"1.10.3"}
```

Comment:
The detail is here: https://github.com/kubernetes/kubernetes/issues/19720#issuecomment-178910637